cellprofiler/modules/measuregranularity.py

```diff
@@ -78,66 +80,71 @@ def add_image(self, can_remove=True):
-            &lt;1 to speed up the processing. Down sampling the image will let you detect larger
-            structures with a smaller sized structure element. A factor &gt;1 might increase the accuracy
-            but also require more processing time. Images are typically of higher resolution than is
-            required for granularity measurements, so the default value is 0.25. For low-resolution images,
-            increase the subsampling fraction; for high-resolution images, decrease the subsampling
-            fraction. Subsampling by 1/4 reduces computation time by (1/4)<sup>3</sup> because the size
-            of the image is (1/4)<sup>2</sup> of original and the range of granular spectrum can
-            be 1/4 of original. Moreover, the results are sometimes actually a little better
-            with subsampling, which is probably because with subsampling the
-            individual granular spectrum components can be used as features, whereas
-            without subsampling a feature should be a sum of several adjacent
-            granular spectrum components. The recommendation on the numerical value
-            cannot be determined in advance; an analysis as in this reference may be
-            required before running the whole set.
-            See this <a href="http://www.ravkin.net/presentations/Statistical%20properties%20of%20algorithms%20for%20analysis%20of%20cell%20images.pdf">
-            pdf</a>, slides 27-31, 49-50.'''))
+                0.25, minval=np.finfo(float).eps, maxval=1, doc="""\
+If the textures of interest are larger than a few pixels, we recommend
+you subsample the image with a factor <1 to speed up the processing.
+Down sampling the image will let you detect larger structures with a
+smaller sized structure element. A factor >1 might increase the accuracy
+but also require more processing time. Images are typically of higher
+resolution than is required for granularity measurements, so the default
+value is 0.25. For low-resolution images, increase the subsampling
+fraction; for high-resolution images, decrease the subsampling fraction.
+Subsampling by 1/4 reduces computation time by (1/4):sup:`3` because the
+size of the image is (1/4):sup:`2` of original and the range of granular
+spectrum can be 1/4 of original. Moreover, the results are sometimes
+actually a little better with subsampling, which is probably because
+with subsampling the individual granular spectrum components can be used
+as features, whereas without subsampling a feature should be a sum of
+several adjacent granular spectrum components. The recommendation on the
+numerical value cannot be determined in advance; an analysis as in this
+reference may be required before running the whole set. See this `pdf`_,
+slides 27-31, 49-50.
+
+.. _pdf: http://www.ravkin.net/presentations/Statistical%20properties%20of%20algorithms%20for%20analysis%20of%20cell%20images.pdf
+pandoc 1.19.2
```

Comment:
Remove this and the next couple lines

Reply:
Starting and ending where? The entire line starting with "See this pdf..." and then the link and reference should be removed as well?

Reply:
The "+ pandoc" through the copyright 2 lines below- that's not part of the original text, I think it was a copy-paste error from the bottom of the pandoc conversion screen.

Reply:
Oh, yes, definitely; that makes sense :)